recipe-server/client/control_new/less/ant-override.less

```diff
@@ -20,3 +20,10 @@
 .ant-menu-submenu-title {
   background: rgba(0, 0, 0, 0.025);
 }
+
+// ant Card's `noHovering` doesn't seem to do anything, so this class makes up for it.
```

Comment:
I believe this is just until we upgrade Ant. I'd rather not include this and this will get dealt with once we upgrade Ant (which should happen shortly after this branch merges).